C99 error on CentOS 7

Seeing the following on CentOS 7 (CentOS Linux release 7.7.1908) when I run `make` as per instructions from [here](https://github.com/lxc/lxcfs#building). 

```
bindings.c: In function 'prune_initpid_store':
bindings.c:230:2: error: 'for' loop initial declarations are only allowed in C99 mode
  for (int i = 0; i < PIDNS_HASH_SIZE; i++) {
  ^
bindings.c:230:2: note: use option -std=c99 or -std=gnu99 to compile your code
bindings.c:231:3: error: 'for' loop initial declarations are only allowed in C99 mode
   for (struct pidns_init_store *entry = pidns_hash_table[i], *prev = NULL; entry;) {
   ^
bindings.c: In function 'cgfs_mount_hierarchies':
bindings.c:694:2: error: 'for' loop initial declarations are only allowed in C99 mode
  for (struct hierarchy **h = cgroup_ops->hierarchies; h && *h; h++) {
  ^
bindings.c: In function 'lxcfs_init':
bindings.c:759:2: error: 'for' loop initial declarations are only allowed in C99 mode
  for (struct hierarchy **h = cgroup_ops->hierarchies; h && *h; h++, i++) {
  ^
```
I tried adding a `-std=c99` as per [this](https://stackoverflow.com/a/29338269/3261361), but this did not work. 

Any suggestions how to fix this?

--- a/configure.ac
+++ b/configure.ac
@@ -248,8 +248,8 @@ AX_CHECK_COMPILE_FLAG([-Wnested-externs], [CFLAGS="$CFLAGS -Wnested-externs"],,[
 AX_CHECK_COMPILE_FLAG([-fasynchronous-unwind-tables], [CFLAGS="$CFLAGS -fasynchronous-unwind-tables"],,[-Werror])
 AX_CHECK_COMPILE_FLAG([-pipe], [CFLAGS="$CFLAGS -pipe"],,[-Werror])
 AX_CHECK_COMPILE_FLAG([-fexceptions], [CFLAGS="$CFLAGS -fexceptions"],,[-Werror])
-CFLAGS="$CFLAGS -fvisibility=hidden"
 
+CFLAGS="$CFLAGS -fvisibility=hidden -Wvla -std=gnu11"
 AX_CHECK_LINK_FLAG([-z relro], [LDFLAGS="$LDFLAGS -z relro"],,[])
 AX_CHECK_LINK_FLAG([-z now], [LDFLAGS="$LDFLAGS -z now"],,[])
 

--- a/src/proc_cpuview.c
+++ b/src/proc_cpuview.c
@@ -433,7 +433,7 @@ static bool read_cpu_cfs_param(const char *cg, const char *param, int64_t *value
 	if (!cgroup_ops->get(cgroup_ops, "cpu", cg, file, &str))
 		return false;
 
-	if (sscanf(str, first ? "%"PRId64 : "%*"PRId64" %"PRId64, value) != 1)
+	if (sscanf(str, first ? "%" PRId64 : "%*" PRId64 " %" PRId64, value) != 1)
 		return false;
 
 	return true;

